Engine/src/Console/CVarUtil.bf

```diff
@@ -0,0 +1,182 @@
+using System;
+
+namespace SteelEngine.Console
+{
+	static class CVarUtil
+	{
+		public static bool TryParse(CVar cvar, Span<StringView> args, ref bool result, out bool didChange)
+		{
+			let currentVal = result;
+			switch (args[0])
+			{
+			case "true", "True", "TRUE", "1":
+				result = true;
+			case "false", "False", "FALSE", "0":
+				result = false;
+			default:
+				return false;
```

Comment:
In most languages, all non-zero numbers are truthy. Should we do the same here rather than default to false?